samples/bot-receive-channel-messages-withRSC/nodejs/Readme.md

```diff
@@ -0,0 +1,105 @@
+﻿Using this Node JS sample, a bot can receive all channel messages with RSC without @mention.
+For reference please check [Receive Channel messages with RSC](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/channel-messages-with-rsc)
+
+This feature shown in this sample is currently available in Public Developer Preview only.
+
+## Key features
+
+- Showing messages based on option selected
+
+![Channel messages](images/channel_messages.png)
```

Comment:
Let's include the screenshot for installation screen which shows the additional permissions.

Reply:
Updated witth screenshots